apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/DatabaseManagerImpl.java

```diff
@@ -194,6 +194,16 @@ public TransactionalDataSource createAndAddShard(Long shardId, DbVersion dbVersi
         return dataSources;
     }
 
+    @Override
+    public int closeAllShardDataSources() {
+        int closedDatasources = 0;
+        for (TransactionalDataSource dataSource : connectedShardDataSourceMap.values()) {
+            dataSource.shutdown();
+            closedDatasources++;
+        }
+        return closedDatasources;
```

Comment:
put debug logging here pls. How many DS were closed ?

Reply:
Its not important for fullReset